lib/git.js

```diff
@@ -64,7 +64,7 @@ async function finishSprint(config) {
       }
       const commitMessage = commit.substring(commit.indexOf(' ') + 1);
       if (commitMessage.indexOf('Merge pull request') !== -1) {
-        const issue = toIssueNumberAndName(commitMessage.substring(commitMessage.lastIndexOf(' ') + 1), owner + '/');
+        const issue = toIssueNumberAndName(commitMessage.substring(commitMessage.lastIndexOf('/') + 1));
```

Comment:
`commitMessage.lastIndexOf('/') + 1` is still not failsafe
it really depends on the nature of free input - which is likely not in expected format?

Reply:
Format is `${projectName}-sprint-${sprintNumber}/${issueNumber}-${issueName}` and we don't expect `${issueName}` to contain another `/`

Reply:
hmm but you are using commit msg?

Reply:
Merge commit message. Somethinkg like this
![image](https://user-images.githubusercontent.com/13611391/59647819-ef7c2500-91a6-11e9-81ec-9f5497b4cc6d.png)

Reply:
for this screenshot, what will be the output? `1848-add-linting` ?

Reply:
```
{
  issueNumber: 1848,
  issueName: 'add linting'
}
```